src/Psalm/Internal/CliUtils.php

```diff
@@ -147,8 +147,13 @@ public static function requireAutoloaders(
             exit(1);
         }
 
-        define('PSALM_VERSION', Versions::getVersion('vimeo/psalm'));
-        define('PHP_PARSER_VERSION', Versions::getVersion('nikic/php-parser'));
+        try {
```

Comment:
Please add $version = null; just above the try. Psalm does not like working with possibly undefined variables